arrows/mvg/algo/initialize_cameras_landmarks.cxx

```diff
@@ -1,4 +1,4 @@
-/*ckwg +29
+/*cOAkwg +29
```

Comment:
looks like some extra character were added by mistake

Reply:
This is now fixed